vault/dendron.tutorial.taking-notes.md

```diff
@@ -2,53 +2,68 @@
 id: 4RFQMNA91cRW12VJl_PFd
 title: Taking Notes
 desc: Taking Notes
-updated: 1636527192313
+updated: 1639196680642
 created: 1625563944736
 nav_order: 1
 ---
 
 ### Create a Note
 
-To create a note, use `Ctrl+L` / `Cmd+L` to bring up Dendron's lookup. This is a shortcut for the `Dendron: Lookup` command.
+To create a note, use `Ctrl+L` / `Cmd+L` to bring up [[Dendron's Lookup|dendron://dendron.dendron-site/dendron.topic.lookup]]. This is a shortcut for the `Dendron: Lookup` command.
 
 > 💡 The lookup command is the main way to interact with Dendron. It is used for both looking up your notes and creating new notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you. Remember the `Ctrl+L` / `Cmd+L` shortcut.
 
-> 🌱 Try it yourself - bring up the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit Enter. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note.
+> 🌱 Try it yourself! Open the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit `Enter`. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note you were viewing.
 
-This should have created a note named `recipes.md`. Notes in Dendron are just plaintext markdown. They live in your file system and are portable across any platform, which makes it very easy to import/ export your notes and even to publish your notes to a hosted website. You can try to add some content to the body of your `recipes` note now.
+This should have created a note named `recipes.md`. Notes in Dendron are just plaintext [[markdown|dendron://dendron.dendron-site/dendron.topic.markdown]]. They live in your file system and are portable across any platform, which makes it very easy to import/export your notes and even to [[publish your notes to a hosted website|dendron://dendron.dendron-site/dendron.topic.publish]]. You can try to add some content to the body of your `recipes` note now.
 
-The --- section at the top of each note is frontmatter. Frontmatter are custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing. Don't modify the id attribute on the front matter.
+The `---` section at the top of each note is **frontmatter**. [[Frontmatter|dendron://dendron.dendron-site/dendron.topic.frontmatter]] contains custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing.
```

Comment:
this would benefit from a screenshot. link frontmatter to [[Frontmatter|dendron.topic.frontmatter]]

Reply:
Went with `yaml` code block instead of screenshot